types/angular-bootstrap-lightbox/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for angular-bootstrap-lightbox
+// Type definitions for angular-bootstrap-lightbox 0.12.0
```

Comment:
```suggestion
// Type definitions for angular-bootstrap-lightbox 0.12
```
https://github.com/definitelyTyped/DefinitelyTyped/#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

Reply:
@patarapolw Thank you. I have updated the code.